### PR TITLE
Mention syntax highlighting, and why we install an editor in the first place

### PR DESCRIPTION
**Pull request by hjwp**
Some tweaks suggested by @HonzaKral -- explain syntax highlighting and why it's useful, the first time we use an editor in the python intro chapter.

Also a brief discussion of why we need a text editor, in the editor installation chapter

### CODE REVIEW

File: en/code_editor/README.md [hunk 1]
```diff
@@ -23,3 +23,16 @@ Sublime Text is a very popular editor with a free evaluation period. It's easy t
...
+
+
+
+# Why are we installing a code editor?
+
+You might be wondering why we are installing this special code editor software, rather than using something like Word or Notepad.
+
+The first is that code needs to be **plain text**, and the problem with programs like Word and Textedit is that they don't actually produce plain text, they produce rich text (with fonts and formatting), using custom formats like rtf.
+
+The second reason is that code editors are specialised in editing code, so they can provide helpful features, like syntax highlighting, or automatically closing quotes for you.
```
Comment (bmispelon) [inline en/code_editor/README.md]: Isn't "syntax highlighting" defined only in a later chapter?

I would either drop it or at least explain what that means in a few words.

**Comment by bmispelon** [comment]
Looks good to me, except for that one bit about syntax highlighting.

:frog:

**Comment by hjwp** [comment]
how 'bout that?

**Comment by bmispelon** [comment]
Perfect, thanks! :ant:

**Pull request merged**